cmd/operator-sdk/run/local.go

```diff
@@ -52,9 +53,15 @@ type runLocalArgs struct {
 func (c *runLocalArgs) addToFlags(fs *pflag.FlagSet) {
 	prefix := "[local only] "
 
-	fs.StringVar(&c.watchNamespace, "watch-namespace", "",
-		prefix+"The namespace where the operator watches for changes. Set \"\" for AllNamespaces, "+
-			"set \"ns1,ns2\" for MultiNamespace")
+	// Currently (plugin phase 1), the new layout is not customized and has not the WATCH_NAMESPACE env var scaffolded
+	// by default into in; the main.go and manager.yaml as it is not importing sdk into go.mod. So, was decide that
+	// for now at least we would not allow the watch-namespace flag.
```

Comment:
Just a nit on rephrasing this comment to make it clearer:
```suggestion
	// The main.go and manager.yaml scaffolds in the new layout do not support the WATCH_NAMESPACE
	// env var to configure the namespace that the operator watches. The default is all namespaces.
	// So this flag is unsupported for the new layout.
```